ci/prow/config_start.yaml

```diff
@@ -236,26 +236,66 @@ metadata:
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
+  namespace: default
   name: "sinker"
 rules:
   - apiGroups:
-      - ""
+      - "prow.k8s.io"
     resources:
-      - pods
+      - prowjobs
     verbs:
       - delete
       - list
+      - watch
+      - get
   - apiGroups:
-      - "prow.k8s.io"
+      - ""
     resources:
-      - prowjobs
+      - configmaps
+    resourceNames:
+      - prow-sinker-leaderlock
```

Comment:
Which resource is named prow-sinker-leaderlock? (Maybe I'm missing some context here)

Reply:
I didn't investigate what that is, I simply copied from the main Prow config. I'm really doubtful about deeply understanding and documenting all the stuff copied from the main Prow config, since we don't follow its development too closely.